From be4884c74355e0b84552e95a5ea2a4a808693acb Mon Sep 17 00:00:00 2001
From: Zamir SUN <sztsian@gmail.com>
Date: Mon, 12 Oct 2020 21:05:29 +0800
Subject: [PATCH] trace-cmd: Temporary move libtraceevent back to
 %{_libdir}/trace-cmd

Signed-off-by: Zamir SUN <sztsian@gmail.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index b034042..fc0de09 100644
--- a/Makefile
+++ b/Makefile
@@ -68,7 +68,7 @@ plugin_tracecmd_dir = $(libdir)/trace-cmd/plugins
 python_dir ?= $(libdir)/traceevent/python
 var_dir = $(HOME)/.trace-cmd/
 else
-plugin_traceevent_dir = $(libdir)/traceevent/plugins
+plugin_traceevent_dir = $(libdir)/trace-cmd/plugins
 python_dir ?= $(libdir)/trace-cmd/python
 PLUGIN_DIR_TRACEEVENT = -DPLUGIN_TRACEEVENT_DIR="$(plugin_traceevent_dir)"
 PLUGIN_DIR_TRACECMD = -DPLUGIN_TRACECMD_DIR="$(plugin_tracecmd_dir)"
-- 
2.26.2

