diff -up ./Makefile.orig ./Makefile
--- ./Makefile.orig	2016-08-18 00:26:57.580940826 -0400
+++ ./Makefile	2016-08-18 00:29:51.594273746 -0400
@@ -31,6 +31,13 @@ $(call allow-override,AR,$(CROSS_COMPILE
 EXT = -std=gnu99
 INSTALL = install
 
+# figure out what arch we are on and install to the right place
+ARCH = $(shell getconf LONG_BIT)
+LIBDIR_32 = lib
+LIBDIR_64 = lib64
+libdir=$(LIBDIR_$(ARCH))
+
+
 # Use DESTDIR for installing into a different root directory.
 # This is useful for building a package. The program will be
 # installed in this directory as if it was the root directory.
