diff -up trace-cmd-2.1.0/Makefile.orig trace-cmd-2.1.0/Makefile
--- trace-cmd-2.1.0/Makefile.orig	2013-02-13 10:50:10.355656277 -0500
+++ trace-cmd-2.1.0/Makefile	2013-02-13 10:51:37.409584197 -0500
@@ -31,6 +31,12 @@ $(call allow-override,AR,$(CROSS_COMPILE
 EXT = -std=gnu99
 INSTALL = install
 
+# figure out what arch we are on and install to the right place
+ARCH = $(shell getconf LONG_BIT)
+LIBDIR_32 = lib
+LIBDIR_64 = lib64
+LIBDIR=$(LIBDIR_$(ARCH))
+
 # Use DESTDIR for installing into a different root directory.
 # This is useful for building a package. The program will be
 # installed in this directory as if it was the root directory.
@@ -56,7 +62,7 @@ ifeq ($(prefix),$(HOME))
 plugin_dir = $(HOME)/.trace-cmd/plugins
 python_dir = $(HOME)/.trace-cmd/python
 else
-plugin_dir = $(prefix)/lib/trace-cmd/plugins
+plugin_dir = $(prefix)/$(LIBDIR)/trace-cmd
 python_dir = $(prefix)/lib/trace-cmd/python
 PLUGIN_DIR = -DPLUGIN_DIR="$(plugin_dir)"
 PYTHON_DIR = -DPYTHON_DIR="$(python_dir)"
